tools/builder/rollup.config.js

```diff
@@ -6,22 +6,35 @@ import typescript from "rollup-plugin-typescript2";
 
 import setup from "./setup.cjs";
 
+const baseBuild = {
+  input: "./pkg.ts",
+  plugins: [resolve(), commonjs() /* only for blake2b? */, typescript(), json()],
+  treeshake: {
+    moduleSideEffects: false,
+  },
+  external: ["lmdb", "@matrixai/quic", "node:fs", "node:assert", "node:os"],
+};
+
 export default [
+  // ES Module build
   {
-    input: "./pkg.ts",
+    ...baseBuild,
     output: {
-      file: setup.outFile,
+      file: setup.cjsOutFile,
       format: "cjs",
     },
-    plugins: [resolve(), commonjs() /* only for blake2b? */, typescript(), json()],
-    treeshake: {
-      moduleSideEffects: false,
+  },
+  // Common JS build
+  {
+    ...baseBuild,
+    output: {
+      file: setup.esmOutFile,
+      format: "es",
     },
-    external: ["lmdb", "@matrixai/quic", "node:fs", "node:assert", "node:os"],
   },
   {
     input: setup.typesInput,
-    output: [{ file: setup.outFile.replace(".js", ".d.ts"), format: "es" }],
+    output: [{ file: setup.esmOutFile.replace(".js", ".d.ts"), format: "es" }],
```

Comment:
is it enough to export only one d.ts file? don't we need a separate one for cjs?

Reply:
I think so, yes.